integreat_cms/deepl_api/deepl_api_client.py

```diff
@@ -75,7 +75,7 @@ def get_target_language_key(target_language: Language) -> str:
                 return code
         return ""
 
-    def translate_queryset(
+    def translate_queryset(  # noqa: PLR0915
```

Comment:
Please remove this line after #3549 is merged

Reply:
I should have waited 🙈 

Hmm, depends which PR gets finished first?

Reply:
Alright 😄